### PR TITLE
Adds document info request in unit tests

### PR DESCRIPTION
**Pull request by ChrisJBurns**
- fixes unit tests
- adds request param into the implementation
- additional implementation will be added in #23

### CODE REVIEW

File: document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImpl.java [hunk 1]
```diff
@@ -55,6 +56,6 @@ public DocumentInfo getDocumentInfo() {
      * @return true if abridged, false if not
      */
     private boolean isAbridged(String accountLink) {
-        return accountLink.matches("/transactions/\\d+(\\-\\d+)+/accounts/\\w+?=");
+        return accountLink.matches("/transactions\\/[0-9\\-]+\\/accounts\\/.*");
```
Comment (BSaunders95) [inline document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImpl.java]: You shouldn't need the escape of the `-` inside the block. Intellij tends to complain about this anyway.

Comment (ChrisJBurns) [reply document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImpl.java]: fixed

File: document-generator-accounts/src/test/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImplTest.java [hunk 2]
```diff
@@ -30,6 +32,9 @@
     @Mock
     private TransactionService transactionService;
 
+    private static final String RESOURCE_ID = "091174-913515-326060";
```
Comment (BSaunders95) [inline document-generator-accounts/src/test/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImplTest.java]: Could these be made more generic? Still follow the same regex pattern, but I feel like unit tests should be as 'fake' as possible.

Comment (ChrisJBurns) [reply document-generator-accounts/src/test/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImplTest.java]: Good point however this PR simply addresses the broken tests, any improvement will can be made further on

Comment (BSaunders95) [reply document-generator-accounts/src/test/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImplTest.java]: I'll approve for now, but I'd advise this be done at some point. Even though it has no bearing on the tests, it all feels a little _too_ real.

**Pull request merged**